integration-tests/src/test/java/uk/gov/di/authentication/api/ResetPasswordIntegrationTest.java

```diff
@@ -189,7 +189,7 @@ void shouldUpdatePasswordAndWriteToAccountRecoveryTableWithIfUserHasVerifiedAuth
 
         var response =
                 makeRequest(
-                        Optional.of(new ResetPasswordCompletionRequest(PASSWORD)),
+                        Optional.of(new ResetPasswordCompletionRequest(PASSWORD, false)),
                         constructFrontendHeaders(sessionId),
```

Comment:
Should there be a test case for when `isForcedPasswordReset` is true to verify the audit event `PASSWORD_RESET_INTERVENTION_COMPLETE` is triggered?